spec/features/card_trainer_spec.rb

```diff
@@ -8,8 +8,8 @@
   end
 
   it "card checking out" do
-    visit user_path(@user.id)
-    fill_in "compared_text", with: 'Mom'
+    visit user_path
+    fill_in "compared_text", with: 'mom'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.